tsconfig.json

```diff
@@ -5,6 +5,7 @@
     "lib": ["ES2020", "DOM", "DOM.Iterable"],
     "module": "ESNext",
     "skipLibCheck": true,
+    "types": ["vite/client"],
```

Comment:
Remove this line after merging main into this branch. In the last commit I also solved this in a different way, adding a `vite.d.ts` file inside the src/ directory.